### ISSUE TITLE
Cannot click on the issue list scroll-bar

### ISSUE THREAD

**Issue by photonstorm** [desktop/desktop#16044]
### The problem

This is probably easier to show a short video than it is to explain:

https://user-images.githubusercontent.com/164476/216118271-2b640bc7-e9d6-4fa6-b717-2d7282cb6793.mp4

Here you can see that I'm typing in a commit message. The message fixes an issue, so I type in `Fix #` and it helpfully brings up the pop-up list of issues. However:

1) Trying to drag the vertical scroll bar causes the list to vanish. You can see this in the video (the first thing I try)
2) Using the up/down arrow keys instead of the mouse causes this list to vanish (the second thing I try)

The only way I can scroll the list is to hover over it and use the mouse-wheel (the 3rd attempt in the video)

However, you can see it is responding to my mouse as I hover over the scrollbar. It just vanishes the moment you click before you get a chance to drag.

### Release version

3.1.5

### Operating system

mac os

### Steps to reproduce the behavior

1. Write a comment message including a #
2. Try to scroll the pop-up list using either the cursor keys or the vertical scroll bar

### Additional context

This may happen on Windows as well, but I haven't tested it yet.

I'm using an external keyboard and mouse on the Mac - they are not made by Apple, in case that makes any difference. I don't believe it should, but hey ... Apple.

**Comment by steveward** [comment]
Thanks for the report @photonstorm, I was able to reproduce this on macOS and Windows. Marking it as a bug.

**Comment by cryptictech** [comment]
I can confirm that I am affected by this in version 3.1.5 on Windows 11. I just updated from 3.1.2 where it did not appear broken.

**Comment by cryptictech** [comment]
After spending some time tracing the development branch and building many versions, I believe I've traced this issue to commit 51a5d7cd4f8bc795d1840b4af95ec017945edb9f as building at 0b711717fdff0a295080a567885a215a7c2ba883 allowed me to use the arrow keys.
